data/items.lua

```diff
@@ -166,6 +166,39 @@ function items:ProcessBankContainer()
   end)
 end
 
+---@param data ItemData
+---@param bindType Enum.ItemBind
+---@return Enum.ItemBinding|nil
+function items:GetItemBinding(data, bindType)
```

Comment:
This function does the same thing over and over again on every pattern match -- reduce this.

Reply:
It also looks like this, and the function GetItemBinding from tooltips convert from binding string to enum to string back and forth. Re-evaluate here and reduce to simpler terms if possible.